src/pps/pps.proto

```diff
@@ -612,11 +612,9 @@ message CreatePipelineRequest {
   ResourceSpec sidecar_resource_limits = 11;
   Input input = 12;
   string description = 13;
-  string cache_size = 14;
```

Comment:
Also need to remove these earlier in the file (lines 396 and 619) and that will reveal some other places that are still using these fields.

Reply:
Thank you! Working on those now